src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -76,11 +76,10 @@ export default class CodDate extends Mixins(DateMixin) {
 
   // Local definitions of computed properties for static type checking.
   // Use non-null assertion operator to allow use before assignment.
-  readonly currentDate!: string
```

Comment:
This was already specified by an imported mixin and caused an error to re-declare it here.